admin-portal/src/icons/ALUMLogoIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const ALUMLogoIcon = ({className}: {className?: string}) => {
+  return (
```

Comment:
Reason I used a typescript file here instead of svg is cauz if I just use svg i have to load it inside of the view as an image and doing that creates this whole thing with shadow DOM which prevents me from styling anything inside the svg like colors by using css. 

Using typescript file here uses the `svg` react component instead of plain svg and allows my DOM to directly have SVG instead of images